Buildings/Controls/OBC/ASHRAE/G36/Atomic/VAVMultiZoneSupFan.mo

```diff
@@ -0,0 +1,403 @@
+          textString="uOpeMod"),
+        Text(
+          extent={{-96,42},{-54,22}},
+          lineColor={0,0,127},
+          textString="boxFloRat"),
+        Text(
+          extent={{-96,-16},{-44,-44}},
+          lineColor={0,0,127},
+          textString="uZonPreResReq"),
+        Text(
+          extent={{-96,-70},{-54,-90}},
+          lineColor={0,0,127},
+          textString="ducStaPre"),
+        Text(
+          extent={{54,-60},{96,-80}},
+          lineColor={0,0,127},
+          textString="yFloRat"),
+        Text(
+          extent={{52,10},{94,-10}},
+          lineColor={0,0,127},
+          textString="yFanSpe"),
+        Text(
+          extent={{52,78},{94,58}},
+          lineColor={0,0,127},
+          textString="ySupFan")}),
+  Documentation(info="<html>
+<p>
+This block implements supply fan control of multizone VAV AHU according to 
+ASHRAE guideline G36, PART5.N.1 (Supply fan control).
+</p>
+<p>a. Supply fan start/stop</p>
+<ul>
+<li>Supply fan shall run when system is in the Cool-down, Setup, or Occupied mode</li>
+<li>If there are any VAV-reheat boxes on perimeter zones, supply fan shall also 
+run when system is in Setback or Warmup mode;</li>
+<li>If the AHU does not serve dual duct boxes (<code>duaDucBox=true</code>) or the AHU
+does not have airflow measurement station (<code>airFloMeaSta=false</code>),
+totallize current airflow rate from VAV boxes to a software point.</li>
+</ul>
+<p>b. Static pressure setpoint reset</p>
+Static pressure setpoint shall be reset using Trim-respond logic using following
+parameters as a starting place:
+<table summary=\"summary\" border=\"1\">
+<tr><th> Variable </th> <th> Value </th> <th> Definition </th> </tr>
+<tr><td>Device</td><td>AHU Supply Fan</td> <td>Associated device</td></tr>
+<tr><td><code>SP0</code></td><td><code>120 Pa (0.5 inches)</code></td><td>Initial setpoint</td></tr>
+<tr><td><code>SPmin</code></td><td><code>25 Pa (0.1 inches)</code></td><td>Minimum setpoint</td></tr>
+<tr><td><code>SPmax</code></td><td><code>maxDesPre</code></td><td>Maximum setpoint</td></tr>
+<tr><td><code>Td</code></td><td><code>10 minutes</code></td><td>Delay timer</td></tr>
+<tr><td><code>T</code></td><td><code>2 minutes</code></td><td>Time step</td></tr>
+<tr><td><code>I</code></td><td><code>2</code></td><td>Number of ignored requests</td></tr>
+<tr><td><code>R</code></td><td>Zone static pressure reset requests</td><td>Number of requests</td></tr>
+<tr><td><code>SPtrim</code></td><td><code>-12 Pa (-0.05 inches)</code></td><td>Trim amount</td></tr>
+<tr><td><code>SPres</code></td><td><code>15 Pa (+0.06 inches)</code></td><td>Respond amount</td></tr>
+<tr><td><code>SPres_max</code></td><td><code>32 Pa (+0.13 inches)</code></td><td>Maximum response per time interval</td></tr>
+</table>
+<br/>
+<p>c. Static pressure control</p>
+Supply fan speed is controlled to maintain duct static pressure at setpoint 
+when the fan is proven on. Where the Zone groups served by the system are 
```

Comment:
There is something wrong with the sentence "Where the zone groups...." This is not English.